common/swaglog.cc

```diff
@@ -92,7 +92,7 @@ static void cloudlog_common(int levelnum, const char* filename, int lineno, cons
 
   std::string log_s;
   log_s += (char)levelnum;
-  ((json11::Json)log_j).dump(log_s);
+  log_s = log_j.dump();
```

Comment:
this should fix the test
```suggestion
  log_s += log_j.dump();
```